package.json

```diff
@@ -58,6 +58,7 @@
     "method-override": "^2.3.10",
     "mongoose": "4.7.4",
     "morgan": "1.9.1",
+    "multer": "false1.4.1",
```

Comment:
Run `yarn config set save-prefix ""` so that yarn won't add `false` before version numbers